applications_tests/dem_3d/packing3d_box.prm

```diff
@@ -4,9 +4,9 @@
 # Simulation and IO Control
 #---------------------------------------------------
 subsection simulation control
-  set time step                 			 = 0.00001
-  set step end       					 = 200000
-  set n total              				 = 3000
+  set time step                 			 = 1e-5
+  set step end       					 = 2e5
+  set n total              				 = 1000
```

Comment:
The number of particle should only be controlled by the insertion mechanism. It hsould only appear once. We can discuss this tomorrow and do another pull request for that.